### DISCUSSION TITLE
Add loki maximum lines config provisioning

### DISCUSSION THREAD

**Discussion by sergeyshaykhullin** [grafana/grafana#41776] [Data Sources]
I need a way to set default maximum lines for provisioned Loki datasource

Here:
![image](https://user-images.githubusercontent.com/46970457/142049230-0bb09cf0-4240-4d6d-8f85-2d6d075a302b.png)

Now it is hardcoded if i understand correct
https://github.com/grafana/grafana/blob/fbd8dc59d95d59b1ab2cee5a1f62be96e7a071ef/pkg/tsdb/loki/loki.go#L147

**Comment by ivanahuckova** [comment]
Hello @sergeyshaykhullin ! As you can see [in the documentation](https://grafana.com/docs/grafana/latest/datasources/loki/#configure-the-data-source-with-provisioning), you can customize maxLines. 
![image](https://user-images.githubusercontent.com/30407135/142869206-d23ee68c-0f06-4789-87d7-28390476dcae.png)

>Now it is hardcoded if i understand correct

This is code that is currently used ONLY for alerting where we support only metrics loki queries, therefore line limit is not used there. The code trough which dashboard and explore queries are run is here:

https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/datasource.ts#L151

**Reply by sergeyshaykhullin** [reply]
Thanks!
